Source/Blazorise/Components/Dropdown/DropdownItem.razor

```diff
@@ -1,16 +1,30 @@
 ﻿@namespace Blazorise
 @inherits BaseComponent
-<a @ref="@ElementRef" id="@ElementId" class="@ClassNames" style="@StyleNames" @onmouseup="@ClickHandler" @onmousedown="@EventCallback.Empty" @onmousedown:preventDefault @attributes="@Attributes">
-    @if ( ShowCheckbox )
-    {
-        <Div Display="Blazorise.Display.InlineBlock" @onclick:stopPropagation @onmouseup:stopPropagation>
-            <Check TValue="bool" Value="@(@checked)" ValueChanged="@CheckedChangedHandler" ValueExpression="@checkedExpression" Disabled="@Disabled" @onclick:stopPropagation>
+
+<a @ref="@ElementRef" id="@ElementId" class="@ClassNames" style="@StyleNames" @onmousedown:preventDefault @onmousedown="@EventCallback.Empty" @onclick="@ClickHandler" @attributes="@Attributes">
+    @Core
+</a>
+
+@code {
+    private protected RenderFragment Core =>
+        @<text>
```

Comment:
I understand that `<text>` is an SVG tag and that it will be removed by Blazor from the final HTML. I like the reusability of it, but I still have problems with it. 

What happens if, one day, the Blazor team decides to render it as a valid HTML tag? We will break down the components. That's why I prefer to almost always write it explicitely.

Reply:
The `<text>` is used in SVG, yes. But it is also a razor directive https://learn.microsoft.com/en-us/aspnet/core/mvc/views/razor?view=aspnetcore-9.0#explicit-delimited-transition

> What happens if, one day, the Blazor team decides to render it as a valid HTML tag?

This seems really unrealistic. A lot of breaking changes for no real benefit.

You can have the text tag if you want:

![image](https://github.com/user-attachments/assets/69fb7666-b840-424f-ad8b-264245fdcf1d)
 
resulting in:

![image](https://github.com/user-attachments/assets/1b98c811-01de-4721-809f-50f358afd098)

Reply:
Ah, I didn't see that in the Blazor docs. So they have a special rule for it. In that case, I agree that it is highly unrealistic it will be changed in the future.